### PR TITLE
[test-only] ApiTest. test for filter appRoleAssigment

### PR DESCRIPTION
**Pull request by ScharfViktor**
Added test for:
- [x] admin gets users with certain role
- [x] admin gets users with certain role and from certain group
- [x] non-admin cannot get users with certain role and from certain group

### CODE REVIEW

**Review by SagarGi — COMMENTED** [review]
Looks Good to me :+1:

**Review by saw-jan — COMMENTED** [review]
LGTM

**Review by saw-jan — APPROVED** [review]

File: tests/acceptance/features/apiGraph/getUser.feature [hunk 1]
```diff
@@ -123,3 +123,54 @@ Feature: get users
       | displayName  | id        | mail              | onPremisesSamAccountName |
       | Alice Hansen | %uuid_v4% | alice@example.org | Alice                    |
       | Carol King   | %uuid_v4% | carol@example.org | Carol                    |
+
+
+  Scenario Outline: non admin user tries to gets users of certain groups
```
Comment (SwikritiT) [inline tests/acceptance/features/apiGraph/getUser.feature:128]:
```suggestion
  Scenario Outline: non-admin user tries to get users of certain groups
```

File: tests/acceptance/features/apiGraph/getUser.feature [hunk 2]
```diff
@@ -123,3 +123,54 @@ Feature: get users
...
+    Then the HTTP status code should be "401"
+    And the last response should be an unauthorized response
+    Examples:
+      | role        |
+      | Space Admin |
+      | User        |
+      | Guest       |
+
+
+  Scenario: admin user gets all users with certain role and members of a certain group
```
Comment (SwikritiT) [inline tests/acceptance/features/apiGraph/getUser.feature:142]:
```suggestion
  Scenario: admin user gets all users with certain roles and members of a certain group
```

File: tests/acceptance/features/apiGraph/getUser.feature [hunk 3]
```diff
@@ -123,3 +123,54 @@ Feature: get users
...
+    Then the HTTP status code should be "200"
+    And the API response should contain following users with the information:
+      | displayName  | id        | mail              | onPremisesSamAccountName |
+      | Brian Murphy | %uuid_v4% | brian@example.org | Brian                    |
+    But the API response should not contain following user with the information:
+      | displayName | id        | mail              | onPremisesSamAccountName |
+      | Carol King  | %uuid_v4% | carol@example.org | Carol                    |
+
+
+  Scenario Outline: non admin user tries to gets users with certain role
```
Comment (SwikritiT) [inline tests/acceptance/features/apiGraph/getUser.feature:167]:
```suggestion
  Scenario Outline: non-admin user tries to get users with a certain role
```

**Pull request merged**